pkgs/misc/emulators/yuzu-mainline/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, mkDerivation, fetchgit
+, cmake, libressl, SDL2, qtbase, python2, alsaLib
+, useVulkan ? true, vulkan-loader, vulkan-headers
+}:
+
+mkDerivation rec {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  # Submodules
+  src = fetchgit {
+    url = "https://github.com/yuzu-emu/yuzu-mainline";
+    rev = "20c42dc9ea8bdc208d9d274350af2ff347876c76";
+    sha256 = "0s9p5di9kcr8sh6iipv21b74c4dhn37an4khy4y7b9s9y8p1haim";
+  };
+
+  nativeBuildInputs = [ cmake libressl ];
+  buildInputs = [ SDL2 qtbase python2 alsaLib ]
+  ++ stdenv.lib.optionals useVulkan [ vulkan-loader vulkan-headers ];
+  cmakeFlags = stdenv.lib.optionals (!useVulkan) [ "DENABLE_VULKAN=No" ];
```

Comment:
Forgot a hyphen:
  `cmakeFlags = stdenv.lib.optionals (!useVulkan) [ "-DENABLE_VULKAN=No" ];`

Reply:
I thought it adds that automatically? I wasn't using one before when i had to use this because of an issue on yuzu's end either, that worked perfectly fine.

Reply:
I tested building with vulkan disabled and it errored, that's how I caught the problem.

Once corrected it built correctly and functions correctly, the renderer toggle on the bottom left of the window is stuck on OpenGL and can't be toggled which proves it's working.